### ISSUE TITLE
object OAuthGrantType is not a member of package scalaoauth2.provider

### ISSUE THREAD

**Issue by berardino** [nulab/scala-oauth2-provider#74]
I have cloned the play2.4 example project https://github.com/tsuyoshizawa/scala-oauth2-provider-example-skinny-orm and created  a custom token endpoint, but it does not compile. I am getting
the error _object OAuthGrantType is not a member of package scalaoauth2.provider_ .

This happens when compiling the project in eclipse as well as from the shell with sbt compile.
The strange thing is that in an other project where I was using spray , I was able to implement
and successfully compile a custom endpoint.
So seems to be related somehow to play ?
Any suggestion ?

```scala
package controllers

import scalaoauth2.provider.TokenEndpoint
import scalaoauth2.provider.Password
import scalaoauth2.provider.OAuthGrantType._

class CustomTokenEndpoint extends TokenEndpoint {

  val passwordNoCred = new Password() {
    override def clientCredentialRequired = false
  }

  override val handlers = Map(
    PASSWORD -> passwordNoCred)
}
```

**Comment by tsuyoshizawa** [comment]
This problem was reproduced in my environment.
I fixed this problem.

Me too. I'm not sure when we can't compile.
Thanks for the report.

**Comment by tsuyoshizawa** [comment]
Backport for 0.15.x branch.
I just published to maven central repository as 0.15.1.
